app/scripts/modules/ecs/src/serverGroup/configure/wizard/networking/Networking.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from 'react';
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+import { IEcsServerGroupCommand } from '../../serverGroupConfiguration.service';
+import { HelpField, TetheredSelect, withErrorBoundary } from '@spinnaker/core';
+import { Alert } from 'react-bootstrap';
+import { Option } from 'react-select';
+
+export interface INetworkingProps {
+  command: IEcsServerGroupCommand;
+  notifyAngular: (key: string, value: any) => void;
+  configureCommand: (query: string) => PromiseLike<void>;
+}
+
+interface INetworkingState {
+  associatePublicIpAddress: boolean;
+  networkMode: string;
+  networkModesAvailable: string[];
+  securityGroups: string[];
+  securityGroupsAvailable: string[];
+  subnetType: string;
+  subnetTypesAvailable: string[];
+}
+
+export class Networking extends React.Component<INetworkingProps, INetworkingState> {
```

Comment:
I think the react components share a global namespace in the running app (i.e., there couldn't be a core `NETWORKING_REACT` component if we exported it with this name), so can we continue to preface this with "ECS"? 

I know we don't do this _everywhere_ ATM, though for things like "task definition" it's ECS specific enough for collisions to be less of a worry.

Reply:
All of our other react components are similar to this, but good callout (we should fix them everywhere though, I'll send out a separate PR to fix the rest).